bin/upgrade

```diff
@@ -118,6 +118,18 @@ function handle_image_upgrade() {
 
   echo "Upgrading config/version from $user_image_version to $seed_image_version"
 
+  local docker_compose_override_path="$TOOLKIT_ROOT/config/docker-compose.override.yml"
+  if [ -f "$docker_compose_override_path" ]; then
+     if grep -E '^\s*image: sharelatex/sharelatex.*' "$docker_compose_override_path"; then
```

Comment:
```suggestion
     if grep -q -E '^\s*image: sharelatex/sharelatex.*' "$docker_compose_override_path"; then
```
WDYT about silencing the `grep` output to keep the script output on point?

Reply:
good point! I have committed the suggestion.

Reply:
Thanks!